core/src/main/java/tc/oc/pgm/listeners/MatchAnnouncer.java

```diff
@@ -102,9 +107,19 @@ public void onMatchEnd(final MatchFinishEvent event) {
         }
       }
 
-      viewer.showTitle(
-          title(title, subtitle, Title.Times.of(Duration.ZERO, fromTicks(40), fromTicks(40))));
+      Title.Times titleTimes = Title.Times.of(Duration.ZERO, fromTicks(40), fromTicks(40));
+
+      if (winners.size() > 3)
+        viewer.showTitle(
+            title(
+                translatable(
+                    "broadcast.gameOver.winners", text(winners.size(), NamedTextColor.YELLOW)),
+                subtitle,
+                titleTimes));
+      else viewer.showTitle(title(title, subtitle, titleTimes));
```

Comment:
given only the text itself changes, i'd consider inlining the if:

```suggestion
      viewer.showTitle(
          title(winners.size() <= 3 ? title : translatable("broadcast.gameOver.winners", text(winners.size(), NamedTextColor.YELLOW)),
              subtitle,
              titleTimes));
```

unsure on how much the formatter will murder this one tho